src/opera_disp_tms/generate_sw_disp_tile.py

```diff
@@ -112,21 +112,21 @@ def update_spatiotemporal_reference(
     if not same_ref_date and update_ref_date:
         raise NotImplementedError('Granule reference date does not match frame metadata, this is not yet supported.')
 
-    same_ref_point = np.isclose(in_granule.attrs['reference_point_geo'], frame.reference_point_geo).all()
+    granule_ref_point = np.array(in_granule.attrs['reference_point_eastingnorthing'])
+    golden_ref_point = np.array(frame.reference_point_eastingnorthing)
+    # Reference points should be within a half pixel width (15 m)
+    same_ref_point = np.all(np.abs(granule_ref_point - golden_ref_point) <= 15)
```

Comment:
Do we need this check? If `update_ref_pont=True`, shouldn't it be fine to directly assign the new value?